libraries.json

```diff
@@ -337,12 +332,12 @@
   {
     "github": "Kotlin/kotlinx.atomicfu",
     "category": "Utils",
-    "maven": "https://kotlin.bintray.com/kotlinx/org/jetbrains/kotlinx/atomicfu-native/"
+    "maven": "https://repo1.maven.org/maven2/org/jetbrains/kotlinx/atomicfu-native/"
   },
   {
     "github": "Dominaezzz/kgl",
     "category": "UI",
-    "maven": "https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-core/"
+    "maven": "https://maven.pkg.github.com/Dominaezzz/kgl"
```

Comment:
authorization required. we can't fetch info from github packages

Reply:
How is better to set info about that? https://github.com/Dominaezzz/kgl#usage

Reply:
remove this lib. automatic fetch can't auth

Reply:
removed